clpy/random/__init__.py

```diff
@@ -20,7 +20,7 @@ def bytes(length):
 # from clpy.random.distributions import lognormal  # NOQA
 from clpy.random.distributions import normal  # NOQA
 # from clpy.random.distributions import standard_normal  # NOQA
-# from clpy.random.distributions import uniform  # NOQA
+from clpy.random.distributions import uniform  # NOQA
```

Comment:
I understand test_userkernel uses random.uniform https://github.com/fixstars/clpy/blob/691fdc41901b761d72ce57dab102137b82e4a985/tests/clpy_tests/core_tests/test_userkernel.py#L13-L14 .

Do you know test case for this? I can not find on #109

Reply:
When you implement some functions, you must add tests if possible.

Reply:
If you don't, please create new issue to solve the test or leave some comments on related existing issue.

Reply:
There is no test case for random.uniform.
But random test ( #109 ) is not working now.
So I commented to #109 .

Reply:
> There is no test case for random.uniform

oh... OK, please fix this problem on #109.